clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStatsReporter.java

```diff
@@ -226,6 +231,44 @@ protected void registerStats() {
           "local_broker_to_follower_consumer",
           IngestionStats::getLocalBrokerFollowerConsumerLatencySensor);
       registerLatencySensor("leader_producer_completion", IngestionStats::getLeaderProducerCompletionLatencySensor);
+
+      registerSensor(
+          new IngestionStatsGauge(this, () -> getStats().getBatchProcessingRequest(), 0, BATCH_PROCESSING_REQUEST));
+      registerSensor(
+          new IngestionStatsGauge(
+              this,
+              () -> getStats().getBatchProcessingRequestError(),
+              BATCH_PROCESSING_REQUEST_ERROR));
+      registerSensor(
+          new IngestionStatsGauge(
+              this,
+              () -> getStats().getBatchProcessingRequestRecords(),
+              0,
+              BATCH_PROCESSING_REQUEST_RECORDS));
+      registerSensor(
```

Comment:
Just a comment: This PR register a few more per store metrics which is what we are trying to avoid. Do you think it is important to have all of them? Maybe not _size/ request_ metric?

Reply:
These detailed metrics will help debug the ingestion performance.

Considering we have removed many dead stores, maybe these new metrics can be fine?

Reply:
I can remove the store-level aggregation and only keep version-level metrics for batch processing.

Reply:
The batch-processing metrics don't have the store-level aggregation, just host-level.

Reply:
OK I see. Yeah if this is the case, then it should be ok to me.